src/FSharpVSPowerTools.Logic/Navigation/GoToDefinitionFilter.fs

```diff
@@ -33,6 +28,12 @@ type internal UrlChangeEventArgs(url: string) =
     inherit EventArgs()
     member __.Url = url
 
+[<NoComparisonAttribute; NoEquality>]
+type GoToDefinitionResolution =
```

Comment:
I think a more fine-grained DU (adding `NotFound` case) could simplify the code by removing option values at their usage.

```fsharp
type GoToDefinitionResult =
    | Internal
    | External of IProjectProvider * ParsedInput option * SnapshotSpan * FSharpSymbolUse
    | LoadDirective of string
    | NotFound
```

I rename other union cases to denote where they come from.